src/c++/perf_analyzer/docs/install.md

```diff
@@ -0,0 +1,107 @@
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Recommended Installation Method
+
+## Triton SDK Container
+
+The recommended way to "install" Perf Analyzer is to run the pre-built executable from within the
+Triton SDK docker container available on the [NVIDIA GPU Cloud
+Catalog](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver). As long
+as the SDK container has its network exposed to the address and port of the
+inference server, Perf Analyzer will be able to run.
+
+```bash
+export RELEASE=<yy.mm> # e.g. to use the release from the end of February of 2023, do `export RELEASE=23.02`
+
+docker pull nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
+
+docker run --gpus all --rm -it --net host nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
+
+# inside container
+perf_analyzer -m <model>
+```
+
+# Alternative Installation Methods
+
+- [Pip](#pip)
+- [Build from Source](#build-from-source)
+
+## Pip
+
+```bash
+pip install tritonclient
+
+perf_analyzer -m <model>
+```
+
+**Warning**: If any runtime dependencies are missing, Perf Analyzer will produce
+errors showing which ones are missing. You will need to manually install
+them.
+
+## Build from Source
+
+The Triton SDK container is used for building in so that some build and runtime dependencies are already installed.
+
+```bash
+export RELEASE=<yy.mm> # e.g. to use the release from the end of February of 2023, do `export RELEASE=23.02`
+
+docker pull nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
+
+docker run --gpus all --rm -it --net host nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk
+
+# inside container
+# prep installing newer version of cmake
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null ; apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main'
+
+# install build/runtime dependencies
+apt update ; apt install -y cmake-data=3.21.1-0kitware1ubuntu20.04.1 cmake=3.21.1-0kitware1ubuntu20.04.1 libcurl4-openssl-dev rapidjson-dev
+
+rm -rf client ; git clone --depth 1 https://github.com/triton-inference-server/client
+
+mkdir client/build ; cd client/build
+
+cmake -DTRITON_ENABLE_PERF_ANALYZER=ON ..
+
+make -j8 cc-clients
```

Comment:
why `-j8`?  Won't `-j` by itself use the max number of threads?

Reply:
Theoretically yes, but I was hesitant to put that in an official doc because both Elias and I have had issues where the compiler like crashes the computer or fails to compile because of resource hogging when using max threads. Of course, 8 could be max threads for some user. Not sure what to recommend here. If we leave off `-j`, then it uses one thread, but that's painstakingly slow.

Reply:
could just leave off the -j. If someone is building from source they likely know what a make command is and how to make it faster. Our job is to communicate correct steps here, not the fastest version of those steps.

Reply:
Removed `-j8` 👍